package.json

```diff
@@ -3,15 +3,19 @@
   "version": "4.13.1",
   "description": "JavaScript library for StarkNet",
   "main": "dist/index.js",
-  "module": "dist/index.mjs",
-  "browser": "dist/index.global.js",
+  "module": "dist/esm/index.js",
+  "browser": "dist/iife/index.js",
   "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/esm/index.js",
+      "browser": "./dist/iife/index.js"
+    }
+  },
   "scripts": {
     "prepare": "npm run build && husky install",
-    "build": "tsup && npm run build:esm && npm run build:iife && npm run build:dts",
-    "build:esm": "tsup --clean false --format esm --platform node",
-    "build:iife": "tsup --clean false --format iife --platform browser",
-    "build:dts": "tsup --clean false --dts-only",
+    "build": "tsup",
```

Comment:
This and the `tsup.config.ts` change of running all the build outputs together will result in an unusable `iife` build result. The `ethereum-cryptography` dependency has a `@noble/hashes` sub-dependency which necessitates that the different build outputs are run with different values of the `tsup` `platform` parameter, `node` (the default value) for `cjs` and `esm`, `browser` for `iife`. Building `iife` with the default value will result in the code attempting to import the native `node.js` `crypto` module in the browser which will fail (search for `node_modules/@noble/hashes/crypto` in the `iife` build result to find the relevant section of code).

Once issue #405 is resolved (probably when PR #399 is finalized) the build script can be revisited and unified as a single `tsup` invocation.